tests/qemu-tests/tests/system/test_commands_kernel.py

```diff
@@ -11,11 +11,13 @@ def test_command_kbase():
 
 def test_command_kchecksec():
     res = gdb.execute("kchecksec", to_string=True)
+    assert res is not None  # for F841 warning
```

Comment:
With `to_string=True` I believe it will always be _some string_, like an empty string.

I gues this is probably fine for now until we change this to some specific string assertion (cc/fyi @gsingh93)

Reply:
Yes, `to_string=True` will always return some string, and @peace-maker is even adding that to the gdbstubs.

I think at least `assert res != ''`  would be better here, until we implement a proper test.

Reply:
The gdb.execute return type is merged already so the gdb-stubs can be updated to make the None check unnecessary.